_config.yml

```diff
@@ -34,6 +34,7 @@ remote:
   homedir: "/home"
   user: "yourUsername"
   prompt: "[yourUsername@gra-login1 ~]$"
+  module_python3: "python3"
```

Comment:
This should be configurable, no guarantee it is called python3, or that it brings in MPI4PY (which is now required)

Reply:
For clarity, can we use "python_bin" instead of "module_python3"? `python3` is a binary executable, more than a "module."